addons/mail/static/tests/mock_server/mock_models/mail_notification.js

```diff
@@ -41,7 +41,7 @@ export class MailNotification extends models.ServerModel {
                 ["failure_type", "mail_message_id", "notification_status", "notification_type"],
                 false
             );
-            data.persona = mailDataHelpers.Store.one(
+            data.res_partner_id = mailDataHelpers.Store.one(
                 ResPartner.browse(notification.res_partner_id),
                 makeKwArgs({ fields: ["name"] })
```

Comment:
This is missing `email` compared to python (is it never tested then?)

Reply:
I know, but if email was not returned and the branch was green, it means there was 0 coverage of the email being displayed anywhere in these flows.

Reply:
I adapted an existing test to add the email part. I tested that it correctly crashes with the email missing from the `res_partner_id`.